waspc/src/Wasp/Psl/Ast/Schema.hs

```diff
@@ -0,0 +1,139 @@
+{-# LANGUAGE DeriveDataTypeable #-}
+
+module Wasp.Psl.Ast.Schema where
+
+import Data.Data (Data)
+
```

Comment:
I got a thumbs up but no answer :D.

Reply:
I was give you the thumbs up regarding this bit: 

>  if this is their nomenclature, in that case better to stick to it whatever they decided to name it.

They use `block` as their building _block_, I've since renamed the element to block so that's gone. https://github.com/prisma/prisma-engines/blob/ae0473d812da9ce31f8c62740c937afb9623f2dc/psl/schema-ast/src/parser/datamodel.pest